boards/arm/mps2/board.yml

```diff
@@ -0,0 +1,9 @@
+board:
+  name: mps2
+  vendor: ARM
+  socs:
+  - name: an385
+  - name: an521
+    variants:
+    - name: 'ns'
+      cpucluster: 'cpu0'
```

Comment:
question: are `''` needed?

Reply:
They are not, should probably run a tidy script to remove them, the original hwmv2 PR contained one with with them and one board without